spot_driver/spot_driver/spot_ros2.py

```diff
@@ -1442,6 +1556,272 @@ def handle_set_volume(self, request: SetVolume.Request, response: SetVolume.Resp
             response.message = f"Error: {e}"
             return response
 
+    def handle_list_ptz(self, request: ListPtz.Request, response: ListPtz.Response) -> ListPtz.Response:
+        """Ros service handler for getting descriptions of any ptz"""
+        if self.spot_cam_wrapper is None:
+            response.success = False
+            response.message = "Spot CAM has not been initialized"
+            return response
+
+        try:
+            proto_descriptions = self.spot_cam_wrapper.ptz.list_ptz()
+            descriptions = []
+            for proto_description in proto_descriptions:
+                ros_msg = PtzDescription()
+                conv.convert_proto_to_bosdyn_msgs_ptz_description(proto_description, ros_msg)
+                descriptions.append(ros_msg)
+            response.success = True
```

Comment:
Have we considered wrapping this up in a function as well? It seems that the `response.success = True` and `response.message = "Success"` is repeated a lot as well

Reply:
We could make a decorator, but I think that is beyond the scope of this PR